include/mbgl/gl/drawable_gl.hpp

```diff
@@ -0,0 +1,77 @@
+#pragma once
+
+#include <mbgl/gfx/drawable.hpp>
+#include <mbgl/gfx/draw_mode.hpp>
+#include <mbgl/gl/vertex_array.hpp>
+#include <mbgl/gl/vertex_attribute_gl.hpp>
+#include <mbgl/programs/segment.hpp>
+
+#include <memory>
+
+namespace mbgl {
+
+template <class AttributeList>
+class Segment;
+
+namespace gfx {
+
+class IndexBuffer;
+class UploadPass;
+class VertexBufferResource;
+
+using UniqueVertexBufferResource = std::unique_ptr<VertexBufferResource>;
+
+} // namespace gfx
+
+namespace gl {
+
+class Texture2D;
+class VertexArray;
+
+class DrawableGL : public gfx::Drawable {
```

Comment:
Personally I like the GL postfix because can sometimes be omitted and then it is unclear which one it is (base class, Metal Drawable, GL Drawable??). But on Slack people preferred the namespace approach.

Maybe both like we have now?

Reply:
They really like the namespace approach.  If you really want to do it the other way revive the discussion on Slack and see if you can talk them into it.